FF152 SVGTextPathElement.side (#44313)
mdn/content: files/en-us/web/api/svganimatedenumeration/index.md

The **`SVGAnimatedEnumeration`** interface describes attribute values which are constants from a particular enumeration and which can be animated.

From SVG 2, both `baseVal` and `animVal` reflect the non-animated value of the attribute.
The allowed values of its properties depend on the associated attribute.

## Instance properties

- {{domxref("SVGAnimatedEnumeration.baseVal", "baseVal")}}
  - : An integer that is the base value of the given attribute before applying any animations.
- {{domxref("SVGAnimatedEnumeration.animVal", "animVal")}} {{ReadOnlyInline}}

>   - : This is the same as [baseVal](#baseval) in SVG 2.

Reviewer: ```suggestion
  - : This is the same as [`baseVal`](#baseval) in SVG 2.
```

## Instance methods

The `SVGAnimatedEnumeration` interface does not provide any specific methods.

## Examples

## Basic usage

> Considering this snippet with a {{SVGElement("clipPath")}} element: Its {{SVGAttr("clipPathUnits")}} is associated with a `SVGAnimatedEnumeration` object.

Reviewer: ```suggestion
Consider this snippet with a {{SVGElement("clipPath")}} element: Its {{SVGAttr("clipPathUnits")}} is associated with a `SVGAnimatedEnumeration` object.
```

Reviewer: Same change on the property pages too?

</svg>
```

> This snippet gets the element, and logs the `baseVal` and `animVal` of the {{domxref("SVGClipPathElement.clipPathUnits")}} property.

Reviewer: ```suggestion
The following JavaScript gets the element, and logs the `baseVal` and `animVal` of the {{domxref("SVGClipPathElement.clipPathUnits")}} property.
```

To avoid using "snippet" twice

Reviewer: Same change on the property pages too?
